### PR TITLE
PD-6304 Add Rakefile to release/package props.d

### PR DESCRIPTION
**Pull request by athompson-r7**
This pull requests aims to setup a "oneliner" that can be placed on the build slaves so that we can build packages of props.d.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+task :shrinkwrap => [:install] do
+  sh 'npm shrinkwrap'
+end
+
+task :pack => [:shrinkwrap] do
+  sh 'npm pack'
+end
+
+desc 'Release props.d and prepare to create a release on github.com'
```
Comment (dgreene-r7) [inline Rakefile]: Seeing `props.d` reminds me that we should go through and standardize the naming across the entire project. I'm going to open an issue to address this, but in the meantime, can you change it to `propsd`?

**Comment by dgreene-r7** [comment]
You've got hard-coded project-related strings throughout this file that might be better suited as a task that reads from `package.json`. I'm thinking specifically `props.d` vs `propsd` and how we can centralize everything around the canonical project name.

**Comment by dgreene-r7** [comment]
:+1:

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,100 @@
+require 'json'
+require 'fileutils'
+require 'mkmf'
```
Comment (fmitchell-r7) [inline Rakefile]: Are we using `mkmf` anywhere? I'm not seeing any references, but it might be pulling it helper stuff.

Comment (athompson-r7) [inline Rakefile]: It provides `find_executable`; we are using `find_executable` at line 59.

**Comment by fmitchell-r7** [comment]
:+1: Thanks for getting this all working! :shipit:

**Comment by athompson-r7** [comment]
Thanks guys!

**Pull request merged**